VariablesHomework/VariablesHomework.cs

```diff
@@ -7,51 +7,63 @@ public class Variables
     {
         [TestMethod]
         public void Declaration()
-        {
+
+
+        {   // Declared the variable by giving it a type: int and a identifier: name
+            // initialized  the variable by assigning the value: 25 
+            int number = 25;
             Assert.IsTrue(number == 25); // Make this pass
+            bool boolean = true;
             Assert.IsTrue(boolean == true); // Make this pass
-            Assert.IsTrue(decimalValue == 99.9); // Make this pass
+            double decimalNum = 99.9;
+            Assert.IsTrue(decimalNum == 99.9); // Make this pass
+            float floatingDecimalValue = 25.4f;
             Assert.IsTrue(floatingDecimalValue == 25.4f); // Make this pass
-            Assert.IsTrue(words == "I am a programmer now"); // Make this pass
-            Assert.IsTrue(character == 'z'); // Make this pass
+            string word = "I am a programmer now";
+            Assert.IsTrue(word == "I am a programmer now"); // Make this pass
+            char f = 'z';
+            Assert.IsTrue(f == 'z'); // Make this pass
         }
 
         [TestMethod]
         public void Assignment()
         {
-            int i = 0;
+            int i = 25;
+        
             Assert.IsTrue(i == 25); // Make this pass
-
+            i = 100;
             Assert.IsTrue(i == 100); // Then make this pass
 
-            float f = 0.0f;
-            Assert.IsTrue(f == 3.5); // Make this pass
-
-            Assert.IsTrue(f == 99.9); // Then make this pass
+            float f = 3.5f;
+            Assert.IsTrue(f == 3.5f); // Make this pass
+            f = 99.9f;
+            Assert.IsTrue(f == 99.9f); // Then make this pass
 
-            bool b = false;
+            bool b = true;
             Assert.IsTrue(b); // Make this pass
-
+            b = false;
             Assert.IsTrue(b == false); // Then make this pass
 
-            double d = 0.0;
+            double d = 10.0;
             Assert.IsTrue(d == 10.0); // Make this pass
-
+            d = 20.0;
             Assert.IsTrue(d == 20.0); // Then make this pass
 
 
-            string s = "";
+            string s = "word";
             Assert.IsTrue(s == "word"); // Make this pass
-
+            s = "next";
             Assert.IsTrue(s == "next"); // Then make this pass
 
-            char c = 'a';
+            char c = 'b';
             Assert.IsTrue(c == 'b'); // Make this pass
+            
+            Assert.IsTrue(c != 'c'); // Then make this pass
 
-            Assert.IsTrue(c == 'c'); // Then make this pass
-
+            
             int z = i;
-            Assert.IsTrue(z == /*Answer here*/); // Make this pass
+            z = 25;
```

Comment:
If you just keep `int z = i` and don't change `z`, then what is `z`'s value?

Reply:
25. because the i = 25; integar at the top of this section

Reply:
Is it? Or has `i` changed since then?

Reply:
Oh i = 100;

Reply:
If you stay on this git branch (JaronVariableHomework2_git), you can make your change, then commit and push. This pull request should update with the new commit, without needing a new pull request.

Reply:
I think not keeping the comments that showed where the answers would go messed me up quite a bit when making corrections. Are we good to just leave those in and put the answers to the left of the comments from now on?

Reply:
Like `Assert.IsTrue(someVariable == 42 /*Answer here*/);`? That works.

Reply:
Yeah